src/pages/settings/organization/Credits.vue

```
@@ -0,0 +1,866 @@
+  }
+}
+
+function summarizeTypes(typeCounts: Record<Database['public']['Enums']['credit_transaction_type'], number>) {
+  const entries = Object.entries(typeCounts)
+    .filter(([, count]) => count > 0)
+    .sort((a, b) => b[1] - a[1])
+    .map(([type, count]) => `${transactionLabel(type as Database['public']['Enums']['credit_transaction_type'])} ×${count}`)
+  return entries.join(' • ') || '—'
+}
+
+function summarizeDeductionsByMetric(deductionsByMetric: DailyLedgerRow['deductionsByMetric']) {
```

ESLint

[error] 'summarizeDeductionsByMetric' is defined but never used. Allowed unused vars must match /^_/u.

(unused-imports/no-unused-vars)